pysat/instruments/icon_mighti.py

```diff
@@ -69,69 +141,43 @@ def init(self):
     inst : (pysat.Instrument)
         Instrument class object
 
-    Returns
-    --------
-    Void : (NoneType)
-        modified in-place, as desired.
-
     """
 
-    logger.info("Mission acknowledgements and data restrictions will be printed " +
-          "here when available.")
+    logger.info(mm_icon.ackn_str)
+    self.meta.acknowledgements = mm_icon.ackn_str
+    self.meta.references = ''.join((mm_icon.refs['mission'],
+                                    mm_icon.refs['mighti']))
 
     pass
 
 
-def clean(inst, clean_level=None):
-    """Provides data cleaning based upon clean_level.
-
-    clean_level is set upon Instrument instantiation to
-    one of the following:
-
-    'Clean'
-    'Dusty'
-    'Dirty'
-    'None'
-
-    Routine is called by pysat, and not by the end user directly.
-
-    Parameters
-    -----------
-    inst : (pysat.Instrument)
-        Instrument class object, whose attribute clean_level is used to return
-        the desired level of data selectivity.
-
-    Returns
-    --------
-    Void : (NoneType)
-        data in inst is modified in-place.
-
-    Note
-    ----
-        Supports 'clean', 'dusty', 'dirty', 'none'
+def default(inst):
+    """Default routine to be applied when loading data.  Adjusts epoch timestamps
+    to datetimes and removes variable preambles.
 
     """
 
-    if clean_level != 'none':
-        logger.info("Cleaning actions for ICON MIGHTI aren't yet defined.")
+    mm_gen.convert_timestamp_to_datetime(inst, sec_mult=1.0e-3)
+    if not inst.kwargs['keep_original_names']:
+        remove_preamble(inst)
 
-    return
 
+def remove_preamble(inst):
+    """Removes preambles in variable names"""
 
-def default(inst):
-    """Default routine to be applied when loading data.
-
-    Note
-    ----
-        Removes ICON preamble on variable names.
+    target = {'los_wind_green': 'ICON_L21_',
+              'los_wind_red': 'ICON_L21_',
+              'vector_wind_green': 'ICON_L22_',
+              'vector_wind_red': 'ICON_L22_',
+              'temperature': ['ICON_L1_MIGHTI_{}_'.format(inst.sat_id.upper()),
+                              'ICON_L23_MIGHTI_{}_'.format(inst.sat_id.upper()),
+                              'ICON_L23_']}
+    mm_gen.remove_leading_text(inst, target=target[inst.tag])
```

Comment:
I'm thinking something like this:
```suggestion
    if (inst.tag != 'temperature') or (sys.version_info.major == 3):
        mm_gen.remove_leading_text(inst, target=target[inst.tag])
    else:
        warnings.warn(' '.join(("Python 3 and xarray 0.12.3 or greater",
                                "required for streamlined variable names for",
                                "this instrument.  Defaulting to original",
                                "names.")))

```